src/portal-components/csbc-home-page/style/_ecosystem.scss

```diff
@@ -30,9 +34,12 @@
         padding: 10px;
         &.isSelected,
         &:hover {
-          color: $primary-action-color;
-          background-color: rgba(71, 51, 125, 0.1);
```

Comment:
Changed this to a transparent variant of the primary color. We do this in a similar component in SRC.

Reply:
This is great!